File: Lecture 6/FinalCode - Java/app/src/main/java/com/inovaeg/androidtrainingpharos2020/RatingResponse.java
package com.inovaeg.androidtrainingpharos2020;

import com.google.gson.annotations.SerializedName;

public class RatingResponse {
    private @SerializedName("status_code")
    String statusCode;
    private @SerializedName("status_message")
    String statusMessage;

    public String getStatusCode() {
        return statusCode;
    }

    public String getStatusMessage() {
        return statusMessage;
    }

    public void setStatusCode(String statusCode) {
        this.statusCode = statusCode;
    }

    public void setStatusMessage(String statusMessage) {
        this.statusMessage = statusMessage;
    }
}

File: Lecture 6/FinalCode - Kotlin/app/src/main/java/com/inovaeg/androidtrainingpharos2020/RatingResponse.kt
package com.inovaeg.androidtrainingpharos2020

import com.google.gson.annotations.SerializedName

class RatingResponse {
    @SerializedName("status_code")
    var statusCode: String? = null

    @SerializedName("status_message")
    var statusMessage: String? = null
}